src/luarocks/manif/writer.lua

```diff
@@ -288,8 +288,6 @@ function writer.make_namespace_file(name, version, namespace)
    assert(type(name) == "string" and not name:match("/"))
    assert(type(version) == "string")
    assert(type(namespace) == "string" or not namespace)
-   name = util.adjust_name_and_namespace(name, { namespace = namespace })
-   name, namespace = util.split_namespace(name)
```

Comment:
Basad on https://github.com/luarocks/luarocks/commit/b6e96a3651db5ed27ec561a110610893e5421a59#diff-2286c8a0ccc4a1cf3148c26d4f2b7e71R287 I think the name argument isn't supposed to have a namespace.